helm/minio/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Multi-Cloud Object Storage
 name: minio
-version: 3.5.7
+version: 3.5.8
```

Comment:
you do not need to change versions @funkypenguin

Reply:
OK, updated to master's current version (3.5.9), hope that was the correct resolution!